src/defs.h

```diff
@@ -29,17 +29,18 @@
 #define ELF_START 0x10000
 #define PTR_SIZE 4
 
-#define DUMP_IR(...)                            \
-    do {                                        \
-        int i;                                  \
-        if (!dump_ir)                           \
-            break;                              \
-                                                \
-        printf("%#010x     ", code_start + pc); \
-        for (i = 0; i < _c_block_level; i++)    \
-            printf("    ");                     \
-                                                \
-        printf(__VA_ARGS__);                    \
+#define DUMP_IR(...)                                                 \
+    do {                                                             \
+        int dump_ir_i;                                               \
```

Comment:
Rename from `dump_ir_i` to `__dump_ir_iter`.

Reply:
Should we avoid using the name with double underscore prefix?

Reply:
> Should we avoid using the name with double underscore prefix?

It is fine for function-like macros.